nav2_waypoint_follower/plugins/photo_at_waypoint.cpp

```diff
@@ -105,7 +110,7 @@ bool PhotoAtWaypoint::processAtWaypoint(
     RCLCPP_INFO(
       logger_,
       "Photo has been taken sucessfully at waypoint %i", curr_waypoint_index);
-  } catch (const cv::Exception & e) {
+  } catch (const std::exception & e) {
```

Comment:
`std::exception` is better, it will also cover up `cv::Exception` as they are derivatives of `std::exception`